### PR TITLE
Tom/update graphing docs

### PR DESCRIPTION
**Pull request by tgberkeley**
Updated the graphs for the first 4 components

### CODE REVIEW

File: docs/library/graphing/area_chart.md [hunk 1]
```diff
@@ -150,6 +165,37 @@ range_area_chart = """rx.area_chart(
...
+            rx.x_axis(
+                data_key="name",
+            ),
+            rx.y_axis(), 
+            rx.legend(),
+            rx.cartesian_grid(
+                stroke_dasharray="3 3",
+            ),
+            data=AreaState.data,
+            width=1000,
```
Comment (Alek99) [inline docs/library/graphing/area_chart.md:194]: Should not set an explicite width/height here so it stays responsive

File: docs/library/graphing/bar_chart.md [hunk 2]
```diff
@@ -149,6 +163,35 @@ range_bar_chart = """rx.bar_chart(
...
+            rx.x_axis(
+                data_key="name",
+            ),
+            rx.y_axis(), 
+            rx.legend(),
+            rx.cartesian_grid(
+                stroke_dasharray="3 3",
+            ),
+            data=BarState.data,
+            width=1000,
```
Comment (Alek99) [inline docs/library/graphing/bar_chart.md:191]: Same here avoid setting an explicit width height or choose to set an explicit height but width should be 100%

File: docs/library/graphing/funnel_chart.md [hunk 3]
```diff
@@ -39,6 +55,18 @@ funnel_chart_example = """rx.funnel_chart(
...
+funnel_chart_example_with_state = """rx.funnel_chart(
+                rx.funnel(
+                    rx.label_list(position="right", data_key="name", fill="#000", stroke="none"),
+                    data_key="value",
+                    data=FunnelState.data,
+                    on_click=FunnelState.randomize_data,
+                ),
+                rx.graphing_tooltip(), 
+                width=730, 
+                height=250)"""
```
Comment (Alek99) [inline docs/library/graphing/funnel_chart.md:68]: Same here width should be set to 100%

**Review by Alek99 — APPROVED** [review]

**Comment by Alek99** [comment]
I'll get my graphing changes in for the min_height logic but same feedback for the comments width should be 100%

File: docs/library/graphing/area_chart.md (hunk 1 above)
Comment (tgberkeley) [reply docs/library/graphing/area_chart.md:194]: changed

File: docs/library/graphing/bar_chart.md (hunk 2 above)
Comment (tgberkeley) [reply docs/library/graphing/bar_chart.md:191]: changed

File: docs/library/graphing/funnel_chart.md (hunk 3 above)
Comment (tgberkeley) [reply docs/library/graphing/funnel_chart.md:68]: changed

**Pull request merged**